monkey/agent_plugins/exploiters/smb/src/smb_options.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import Field
+
+from common.base_models import InfectionMonkeyBaseModel
+
+
+class SMBOptions(InfectionMonkeyBaseModel):
+    agent_binary_upload_timeout: float = Field(
+        default=30.0,
+        gt=0.0,
+        le=100.0,
```

Comment:
These maximums seem somewhat arbitrary. Are they based on anything?

Reply:
Only if we add a note to the description that this could cause the agent to hang on shutdown.

Reply:
Then we should also change things in Hadoop